src/app/models/form.server.model.ts

```diff
@@ -857,6 +858,29 @@ const compileFormModel = (db: Mongoose): IFormModel => {
       .exec()
   }
 
+  FormSchema.statics.updateMsgSrvcName = async function (
+    formId: string,
+    msgSrvcName: string,
+    session?: ClientSession,
+  ) {
+    return session
+      ? this.findByIdAndUpdate(formId, { msgSrvcName }, { session }).exec()
+      : this.findByIdAndUpdate(formId, { msgSrvcName }).exec()
+  }
+
+  FormSchema.statics.deleteMsgSrvcName = async function (
+    formId: string,
+    session?: ClientSession,
```

Comment:
session looks weird here, it should not be the responsibility of this function to care about whether it is in a transaction or not imho

as stated in a different comment, might want to have another function initiating a transaction session and then calling this function